frontend/src/app/hierarchy-viewer/deleteHierarchy.dialog.html

```diff
@@ -0,0 +1,10 @@
+<h1 mat-dialog-title>
+    Delete {{data.name}}{{!data.name.includes('Hierarchy') ? ' Hierarchy' : null}}
```

Comment:
I think it'd look better to keep the title consistent such as "Delete Hierarchy".